types/srcds-rcon/srcds-rcon-tests.ts

```diff
@@ -0,0 +1,16 @@
+import Rcon from "srcds-rcon";
```

Comment:
If you're going to us an ES6 import (vs `import Rcon = require("srcds-rcon");`), you'll need to add either `allowSyntheticDefaultImports` or `esModuleInterop` to your tsconfig.json.